docs/node/full-node/full-node-docker.md

```diff
@@ -0,0 +1,207 @@
+
+## Before you start
+
+The instructions below assume a Linux distribution with `apt` (Debian,
+for example), but you may be able to adapt them to a different
+distribution. Also the Docker images Currently Available are only supported for **Linux/amd64 or x86_64 based** cpus .It's also common for users to run nodes on a cloud
+server. The following list of standard hardware is a recommendation of
+hardware specs that your environment should have:
+
+| Minimum                            | Recommended                         |
+|------------------------------------|-------------------------------------|
+| 4GB RAM                            | 8GB RAM                             |
+| 2 core CPU amd64/x86 architecture  | 4 core CPU  amd64/x86 architecture  |
+| 20-40 GB SSD                       | 200-300 GB SSD                      |
+
+The peer exchange for a Avail full node generally depends on port 30333 being open. When you configure your firewall or security groups for Cloud Providers, make sure these ports are open along with whatever ports you need to access the machine.
+
+#### Info Alternate networks & releases
+
+All instructions below are for the **Kate testnet** network. To join a
+different network, you may need to download a different node version
+from the [node releases
+page](https://github.com/availproject/avail/releases) and a chain
+specification file for the specific network. Contact the Avail team if
+you have questions.
+
+## Useful Links
+
+[Dockerhub Repository](https://hub.docker.com/r/availj/avail/tags)
+
+[Github Releases](https://github.com/availproject/avail/releases)
+
+## Initial Setup
+
+At this point, you should have shell access with root privileges to a linux machine.
+
+### Install Docker 
+Most likely your operating system won’t have Docker installed by default. Please follow the instructions for your particular distribution found here: https://docs.docker.com/engine/install/
+
+We’re following the instructions for Ubuntu. The steps are included below, but please see the official instructions in case they’ve been updated.
+
+```bash
+sudo apt-get update
+sudo apt-get install ca-certificates curl gnupg lsb-release
+sudo mkdir -p /etc/apt/keyrings
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+echo \
+  "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+  $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+sudo apt-get update
+sudo apt-get install docker-ce docker-ce-cli containerd.io docker-compose-plugin
+```
+
+At this point you should have Docker installed.
+
+In many cases, it’s inconvenient to run docker as **`root`** user so we’ll follow the post install steps [here](https://docs.docker.com/engine/install/linux-postinstall/) in order to interact with docker without needing to be **`root:`**
```

Comment:
```suggestion
To avoid running Docker as the **root** user, which is often inconvenient, we'll adhere to the post-installation steps outlined [here](https://docs.docker.com/engine/install/linux-postinstall/). This allows us to interact with Docker without requiring **root** privileges.
```